### ISSUE TITLE
Camera Always at 0,0,0 position

### ISSUE THREAD

**Issue by Devination** [luboslenco/zblend#6]
Hi Lubos, it's been a while since I tried your project (I like the new name by the way, easier to search for) I tried to run the my_project.blend in HTML5 and all I saw was a grey screen. Eventually, I figured out it was the inside of the default cube I was looking at. I had to move the Box a little further out to see it fully.

It seems that the camera will respect Rotation values but the camera is always at 0,0,0 Position when the game runs even if it's moved somewhere else in Blender.

TL;DR:
* Camera's Transform respects Rotation but not Position

Thanks,
Devin
